inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/GroupInfo.java

```diff
@@ -39,6 +40,9 @@ public class GroupInfo implements Serializable {
     @JsonProperty("streams")
     private List<StreamInfo> streams;
 
+    @JsonProperty("extraInfo")
+    private Properties extraInfo;
```

Comment:
Maybe Properties extraInfo -> Map<String,String> properties ?

Reply:
thanks for your advice